positron/components/ModuleLoader.jsm

```diff
@@ -131,20 +144,22 @@ function ModuleLoader(processType) {
     }
 
     if (!file.exists()) {
-      throw new Error('No such module: ' + path);
+      throw new Error(`No such module: ${path} (required by ${requirer.id})`);
     }
 
     // dump('require: module found at ' + uri.spec + '\n');
 
     // Exports provided by the module.
-    let exports = Object.create(null);
+    let exports = Object.create({});
```

Comment:
> What's the reason for using Object.create({}) here (instead of a normal object)?

I'm unsure why I used `Object.create(null)` instead of an object literal when I originally added this line, but the reason I changed it here to make its prototype be an object literal is that https://github.com/mozilla/positron/blob/master/positron/electron/lib/renderer/api/remote.js#L286-L289 calls `__defineGetter__` on the object, and that function isn't defined on it if its prototype is `null` instead of `Object.prototype`.

But there are simpler ways to fix that problem, like setting *exports* to an object literal. In fact it looks like we don't use it outside the *module* object, so it doesn't even need a local variable. I set module.exports directly to an object literal in e12a8c5.